geth/api/backend.go

```diff
@@ -273,13 +264,28 @@ func (b *StatusBackend) registerHandlers() error {
 	if rpcClient == nil {
 		return node.ErrRPCClient
 	}
-	rpcClient.RegisterHandler("eth_accounts", func(context.Context, ...interface{}) (interface{}, error) {
+	rpcClient.RegisterHandler(params.PersonalSignMethodName, func(context.Context, ...interface{}) (interface{}, error) {
+		return nil, errors.New("IGORM: custom erro boi from an override!")
```

Comment:
What does that mean`IGORM: custom erro boi from an override!` ?

Reply:
oops, that is a leftover, this method shouldn't be even present in this PR because it breaks the APIs